packages/shim-deno/src/deno/internal/Conn.ts

```diff
@@ -40,6 +41,38 @@ export class Conn extends FsFile implements Deno.Conn {
   unref(): void {
     this.#socket.unref();
   }
+
+  async read(p: Uint8Array): Promise<number | null> {
+    let wait = false;
+    while (true) {
+      if (wait) {
```

Comment:
What's the purpose of this waiting?

Reply:
So the `read` syscall does not throw EAGAIN.